includes/class-wc-stripe-checkout-sessions-controller.php

```
@@ -134,4 +79,82 @@ public function create_checkout_session(): void {
+	 *
+	 * @return array
+	 */
+	private function build_payment_intent_data(): array {
+		$data     = [];
+		$metadata = [
+			'site_url'     => esc_url_raw( get_site_url() ),
+			'payment_type' => 'single',
+		];
+
+		if ( is_user_logged_in() ) {
+			$wc_customer = WC()->customer;
```

PHPMD (2.15.0)

[error] The variable $wc_customer is not named in camelCase. (undefined)

(CamelCaseVariableName)